### PR TITLE
Shipyard get action test

### PR DESCRIPTION
**Pull request by rupamatt**
Shipyard get action tests

### CODE REVIEW

File: airship_tempest_plugin/tests/api/shipyard/test_actions_ucp.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @decorators.idempotent_id('94901561-7ad1-4e9c-8df8-afe3a7f63c09')
+    def test_list_actions(self):
+        """List of actions, Successful with response status 200"""
+        response, _ = self.shipyard_actions_client.list_actions()
+        self.assertEqual(response['status'], 200)
+
+    @decorators.idempotent_id('b0d4c23a-d3a4-4a12-8e10-ac6f8a98d33e')
+    def test_get_action(self):
+        """Get actions, Successful with response status 200"""
+        response, _ = self.shipyard_actions_client.get_action()
```
Comment (rickbartra91) [inline airship_tempest_plugin/tests/api/shipyard/test_actions_ucp.py:40]: Without passing an action_id, I do not suspect that the response will be 200. Can you confirm?

File: airship_tempest_plugin/tests/api/shipyard/test_actions_ucp.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        self.assertEqual(response['status'], 200)
+
+    @decorators.idempotent_id('6b16f987-8192-4a01-82c4-3ffc1bd9402c')
+    def test_get_action_validation(self):
+        """Get actions validation, Successful with response status 200"""
+        response, _ = self.shipyard_actions_client.get_action_validation()
```
Comment (rickbartra91) [inline airship_tempest_plugin/tests/api/shipyard/test_actions_ucp.py:46]: Without passing an action_id and validation_id, I do not suspect that the response will be 200. Can you confirm?

File: airship_tempest_plugin/tests/api/shipyard/test_actions_ucp.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        self.assertEqual(response['status'], 200)
+
+    @decorators.idempotent_id('a8bc9e6b-bfa3-4635-a1ec-0b9ddc9cb03f')
+    def test_get_action_step(self):
+        """Get actions step, Successful with response status 200"""
+        response, _ = self.shipyard_actions_client.get_action_step()
```
Comment (rickbartra91) [inline airship_tempest_plugin/tests/api/shipyard/test_actions_ucp.py:52]: action_id and step_id likely needed for a successful 200 response

**Review by rickbartra91 — CHANGES_REQUESTED** [review]
As mentioned in the README:

DISCALIMER:
This initial implementation is just to meet the first use case which is RBAC testing. For RBAC testing, we only need to hit the API endpoint and check role permission to the API being tested. Some of the REST clients will need to be rewritten if functional testing is desired. 

In order to actually get a 200 response, I believe you need to provide valid arguments. Additionally, you may need to review the REST client to verify it is using the parameters correctly.

**Comment by rupamatt** [comment]
Started a new pull request for shipyard functional tests.

**Pull request closed without merging**